<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>346</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PAINT</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/icon/Resources/ICON.ico</normaloff>:/icon/Resources/ICON.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="toolsPanelLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SizeConstraint::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QGroupBox" name="brushStyleContainer">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Стиль кисти </string>
        </property>
        <layout class="QVBoxLayout" name="brushStyleLayout">
         <item>
          <widget class="QPushButton" name="normalBtn">
           <property name="text">
            <string>Круглая</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icon/Resources/ICONS-01.png</normaloff>:/icon/Resources/ICONS-01.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">brushStyleGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sprayBtn">
           <property name="text">
            <string>Баллончик</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icon/Resources/ICONS-02.png</normaloff>:/icon/Resources/ICONS-02.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">brushStyleGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="neonBtn">
           <property name="text">
            <string>Неон </string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icon/Resources/ICONS-03.png</normaloff>:/icon/Resources/ICONS-03.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">brushStyleGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="toolsContainer">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Инструмент</string>
        </property>
        <layout class="QVBoxLayout" name="toolsLayout">
         <item>
          <widget class="QPushButton" name="brushBtn">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Кисть</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icon/Resources/ICONS-07.png</normaloff>:/icon/Resources/ICONS-07.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">drawModeGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="lineBtn">
           <property name="text">
            <string>Линия </string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icon/Resources/ICONS-04.png</normaloff>:/icon/Resources/ICONS-04.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">drawModeGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="rectBtn">
           <property name="text">
            <string>Прямоугольник</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icon/Resources/ICONS-06.png</normaloff>:/icon/Resources/ICONS-06.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">drawModeGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ellipseBtn">
           <property name="text">
            <string>Эллипс</string>
           </property>
           <property name="icon">
            <iconset resource="Resources.qrc">
             <normaloff>:/icon/Resources/ICONS-05.png</normaloff>:/icon/Resources/ICONS-05.png</iconset>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">drawModeGroup</string>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="tool_fill">
           <property name="text">
            <string>Заливка</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">drawModeGroup</string>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="colorSelectGroup">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Цвет</string>
        </property>
        <layout class="QVBoxLayout" name="colorLayout">
         <item>
          <layout class="QHBoxLayout" name="basicColorsLayout">
           <item>
            <widget class="QPushButton" name="color_red">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: red;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorSelectGroup1</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="color_blue">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: blue;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorSelectGroup1</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="color_green">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: green;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorSelectGroup1</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="color_yellow">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: yellow;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorSelectGroup1</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="color_black">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: black;</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorSelectGroup1</string>
             </attribute>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="color_custom">
             <property name="text">
              <string>Другой</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <attribute name="buttonGroup">
              <string notr="true">colorSelectGroup1</string>
             </attribute>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="colorHistoryLayout">
           <item>
            <widget class="QPushButton" name="historyBtn1">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="historyBtn2">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="historyBtn3">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="historyBtn4">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="historyBtn5">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="colorIndicator">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::Box</enum>
           </property>
           <property name="text">
            <string>Выбранный цвет</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="brushSizeGroup">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="title">
         <string>Размер кисти </string>
        </property>
        <layout class="QHBoxLayout" name="brushSizeLayout">
         <item>
          <widget class="QSlider" name="brushSize_slider">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="brushSize_label">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="undoRedoLayout">
        <item>
         <widget class="QPushButton" name="undoBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Назад</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/icon/Resources/ICONS-10.png</normaloff>:/icon/Resources/ICONS-10.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="redoBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Вперед</string>
          </property>
          <property name="icon">
           <iconset resource="Resources.qrc">
            <normaloff>:/icon/Resources/ICONS-11.png</normaloff>:/icon/Resources/ICONS-11.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="clearAll">
        <property name="text">
         <string>Очистить холст</string>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/icon/Resources/ICONS-12.png</normaloff>:/icon/Resources/ICONS-12.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="CanvasWidget" name="canvasWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CanvasWidget</class>
   <extends>QWidget</extends>
   <header>canvaswidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="drawModeGroup"/>
  <buttongroup name="brushStyleGroup"/>
  <buttongroup name="colorSelectGroup1"/>
 </buttongroups>
</ui>
